packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -351,3 +357,28 @@ if (bootstrapConfig) {
         Log.error('Sandbox initialization failed: ' + error.message);
     });
 }
+
+/**
+ * Handle higher layer changes when starting UI Adaptation.
+ * When RTA detects higher layer changes an error with Reload triggered text is thrown, the RTA instance is destroyed and the application is reloaded.
+ * For UI5 version lower then 1.84.0 RTA is showing a popup with notification text about the detection of higher layer changes.
+ *
+ * @param error the error thrown when there are higher layer changes when starting UI Adaptation.
+ * @param ui5VersionInfo ui5 version info
+ */
+export async function handleHigherLayerChanges(error: unknown, ui5VersionInfo: Ui5VersionInfo): Promise<void> {
+    const err = getError(error);
+    if (err.message.includes('Reload triggered')) {
+        if (ui5VersionInfo && ui5VersionInfo.minor >= 84 && ui5VersionInfo.major >= 1) {
```

Comment:
Would you be able to use this method to stay consistent:

```suggestion
        if (!isLowerThanMinimalUi5Version(ui5VersionInfo, { major: 1, minor: 84 })) {
```

Reply:
Fixed in [6d6f900](https://github.com/SAP/open-ux-tools/pull/2536/commits/6d6f900315c8f0446b6829c935ec360bf8f506c2)